
string device_model
string robot_conf
string dist_model
string dead_regions

bool is_color
int32 image_height
int32 image_width

float64[12] relative_pose
float64[9] camera_matrix
float64[9] dist_coeffs
float64[9] projection_matrix

